projects/plugins/boost/app/class-jetpack-boost.php

```diff
@@ -273,6 +285,8 @@ public static function activate() {
 	public function deactivate() {
 		do_action( 'jetpack_boost_deactivate' );
 
+		wp_clear_scheduled_hook( 'jetpack_boost_routine_cleanup' );
```

Comment:
The name for the scheduled event is
`jetpack_boost_general_cleanup`
but here it is
`jetpack_boost_routine_cleanup`